### PR TITLE
Landing Event

### PR DESCRIPTION
**Pull request by Jesus-QC**
Added a landing event with a brand new transpiler.

### CODE REVIEW

File: Exiled.Events/EventArgs/LandingEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    /// <summary>
+    /// Contains all the information when a player lands.
+    /// </summary>
+    public class LandingEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LandingEventArgs"/> class.
+        /// </summary>
+        /// <param name="player">The player landing.</param>
```
Comment (NaoUnderscore) [inline Exiled.Events/EventArgs/LandingEventArgs.cs:21]: `<inheritdoc cref="Player"/>`

File: Exiled.Events/EventArgs/LandingEventArgs.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        public LandingEventArgs(Player player) => Player = player;
+
+        /// <summary>
+        /// Gets the player landing.
```
Comment (NaoUnderscore) [inline Exiled.Events/EventArgs/LandingEventArgs.cs:25]: `Gets the <see cref="Exiled.API.Features.Player"/> who's landing.`

File: Exiled.Events/EventArgs/LandingEventArgs.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs
+{
+    using System;
+    using Exiled.API.Features;
+
+    /// <summary>
+    /// Contains all the information when a player lands.
```
Comment (NaoUnderscore) [inline Exiled.Events/EventArgs/LandingEventArgs.cs:14]: `Contains all the information after a <see cref="Exiled.API.Features.Player"/> lands.`

File: Exiled.Events/Handlers/Player.cs [hunk 4]
```diff
@@ -387,6 +387,11 @@ public static class Player
         /// </summary>
         public static event CustomEventHandler<JumpingEventArgs> Jumping;
 
+        /// <summary>
+        /// Invoked when a player lands.
```
Comment (NaoUnderscore) [inline Exiled.Events/Handlers/Player.cs:391]: `Invoked after a <see cref="Exiled.API.Features.Player"/> lands.`

File: Exiled.Events/Handlers/Player.cs [hunk 5]
```diff
@@ -856,6 +861,12 @@ public static class Player
         /// <param name="ev">The <see cref="JumpingEventArgs"/> instance.</param>
         public static void OnJumping(JumpingEventArgs ev) => Jumping.InvokeSafely(ev);
 
+        /// <summary>
+        /// Invoked when a player lands.
```
Comment (NaoUnderscore) [inline Exiled.Events/Handlers/Player.cs:865]: `Called after a <see cref="Exiled.API.Features.Player"/> lands.`

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.Events/Handlers/Player.cs [hunk 6]
```diff
@@ -856,6 +861,12 @@ public static class Player
         /// <param name="ev">The <see cref="JumpingEventArgs"/> instance.</param>
         public static void OnJumping(JumpingEventArgs ev) => Jumping.InvokeSafely(ev);
 
+        /// <summary>
+        /// Called after a <see cref="Exiled.API.Features.Player"/> lands.
```
Comment (NaoUnderscore) [inline Exiled.Events/Handlers/Player.cs:865]: _Called after a player lands._ Is fine

File: Exiled.Events/Handlers/Player.cs [hunk 7]
```diff
@@ -387,6 +387,11 @@ public static class Player
         /// </summary>
         public static event CustomEventHandler<JumpingEventArgs> Jumping;
 
+        /// <summary>
+        /// Invoked after a <see cref="Exiled.API.Features.Player"/> lands.
```
Comment (NaoUnderscore) [inline Exiled.Events/Handlers/Player.cs:391]: _Invoked after a player lands._ Is fine

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/LandingEventArgs.cs [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+    /// <summary>
+    /// Contains all the information after a <see cref="Exiled.API.Features.Player"/> lands.
+    /// </summary>
+    public class LandingEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LandingEventArgs"/> class.
+        /// </summary>
+        /// <param name="player">The <inheritdoc cref="Player"/> landing.</param>
```
Comment (joker-119) [inline Exiled.Events/EventArgs/LandingEventArgs.cs:21]: `<param name="player"><inheritdoc cred="Player"/></param>`

**Review by joker-119 — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/LandingEventArgs.cs (hunk 3 above)
Comment (TheUltiOne) [inline Exiled.Events/EventArgs/LandingEventArgs.cs]: /// Contains all the information when a player lands **on the ground**
It isn't really clear what Landing means, I mean, there aren't much possibilities of it but make it clear.

**Review by TheUltiOne — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/LandingEventArgs.cs (hunk 3 above)
Comment (NaoUnderscore) [reply Exiled.Events/EventArgs/LandingEventArgs.cs]: It's an old version, now it's different.

File: Exiled.Events/EventArgs/LandingEventArgs.cs [hunk 9]
```diff
@@ -18,7 +18,7 @@ public class LandingEventArgs : EventArgs
         /// <summary>
         /// Initializes a new instance of the <see cref="LandingEventArgs"/> class.
         /// </summary>
-        /// <param name="player">The <inheritdoc cref="Player"/> landing.</param>
+        /// <param name="player"><inheritdoc cred="Player"/></param>
```
Comment (joker-119) [inline Exiled.Events/EventArgs/LandingEventArgs.cs:21]: `cref` not `cred`

**Review by joker-119 — CHANGES_REQUESTED** [review]

**Pull request merged**